src/modules/ListMovies.js

```diff
@@ -0,0 +1,55 @@
+export default class ListMovies {
+  constructor() {
+    this.list = [];
+    this.urlApi = 'https://api.tvmaze.com/shows'; // +id
+    this.currentPage = 1;
+  }
+
+  /* Display (Load dynnamically) a limited number of movies */
+  display = () => {
+    this.list.forEach((movie) => {
+      // dynamic load of list of movies
+      const liMovies = document.createElement('li');
+      liMovies.classList.add('flex-center', 'movie');
+      liMovies.id = `movie-${movie.id}`;
+
+      const wrapper = document.createElement('div');
+      const img = document.createElement('img');
+      img.src = movie.image.medium; // `${this.urlApi}${movie.id}/images`
```

Comment:
I know you have spent a lot of time and effort on this project and so forgot some basic mistakes 😄 , Please avoid using inline code as comments
it makes your work look messy and unprofessional. liters don't usually catch these types of mistakes so try to go through your work the search for such mistakes